docs/en/Security.md

```diff
@@ -18,8 +18,8 @@ encryption.
 POSIX permission model is used in Alluxio to assign permissions and
 control access rights.
 
-By default Alluxio runs in non-secure mode in which no authentication and authorization are
-required.
+By default Alluxio runs in `SIMPLE` secure mode in which a simple authentication and authorization
```

Comment:
can you please add more details to clarify `SIMPLE` does not actually do real authentication? meaning that server trusts whoever the client claims to be.